composer.json

```
@@ -24,7 +24,7 @@
         "symfony/mercure-bundle": "^0.3.2",
         "symfony/messenger": "*",
         "symfony/monolog-bundle": "^3.1",
-        "symfony/notifier": "5.2.*",
+        "symfony/notifier": ">=5.4",
```

```suggestion
        "symfony/notifier": "*",
```

Should work as we have `5.4.*` configured below, right?

---

Thanks @xabbuh , I changed it to `*`